### ISSUE TITLE
Narrator Does not Read Out Search Result Content in Npm Install Window

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1359]
## Repo
1. Enable Windows narrator mode.
2. Open the NTVS npm install window.
3. Wait for the window to load fully.
4. Search for `mocha`
5. tab to search results.

![image](https://cloud.githubusercontent.com/assets/12821956/19532295/fb198426-95ef-11e6-9f81-84bb3b9a1cde.png)
## Expected

While navigating through the search results on the left side, the results are read out in some reasonable way, such as by package name.
## Actual

The follow is read:

> Microsoft dot nodejs tools dot npm ui dot read only catalog entry view model;  1 of 6; Selected

I can't find any way to get it to actually read out the package contents from the search results. There are also far more than six results.

**Comment by paulvanbrenk** [comment]
Fixed in Update 3
